### ISSUE TITLE
RSS feed for podcasts

### ISSUE THREAD

**Issue by retrography** [ejurgensen/forked-daapd#880]
Would it be possible to add a feature whereby FDAAPD populates the podcasts section of Remote app by periodically reading the RSS feeds of the added podcasts?

**Comment by chme** [comment]
I would love to see support for this as well. A while ago i did look into this, but never started implementing it (it requires some work). 

Useful could be:
- https://github.com/kstep/libmrss (c library to read rss feeds)
- https://github.com/mlj/castget (a c program that downloads new podcast episodes)

I used castget for a while (periodically executed with a cron job), to automatically download new episodes into a folder in my forked-daapd library. That worked quite well, so maybe that would be an already available solution for you.

**Comment by retrography** [comment]
I will try castget, but I figured probably no download is necessary given that the content can be streamed. Downloading is good strategy only if you are a regular listener.

**Comment by chme** [comment]
I agree, it would be nice if, forked-daapd would directly read the rss feeds and then stream the episodes. Ideally forked-daapd would also offer some basic search functionality for podcasts ...

**Comment by ejurgensen** [comment]
Sounds like a good enhancement - let's have it open here and see if someone develops it.

**Comment by ejurgensen** [comment]
Closing this since it is has since been implemented by @whatdoineed2do
